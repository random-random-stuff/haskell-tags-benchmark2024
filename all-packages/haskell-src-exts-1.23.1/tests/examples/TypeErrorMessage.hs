newtype PoC = PoC (forall b . b)
