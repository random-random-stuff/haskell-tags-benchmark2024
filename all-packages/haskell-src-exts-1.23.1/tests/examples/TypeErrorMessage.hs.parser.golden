ParseFailed
  (SrcLoc "tests/examples/TypeErrorMessage.hs" 2 1)
  "At least one of ExplicitForAll or TypeOperators language extensions needs to be enabled. Please add: {-# LANGUAGE ExplicitForAll #-} or {-# LANGUAGE TypeOperators #-} language pragma at the top of your module."
